src/domain/event/utils.ts

```diff
@@ -1137,6 +1141,10 @@ export const getIsButtonVisible = ({
   }
 };
 
+const validateCreatedBy = (createdBy: string | null | undefined): boolean => {
+  return createdBy !== (null || undefined || '' || ' - ');
```

Comment:
I think we would have the same effect with `return Boolean(createdBy) || createdBy !== ' - '` 🤔